varpivo/api/recipe.py

```diff
@@ -57,7 +70,7 @@ async def get(self, recipeId):
     async def post(self, recipeId):
```

Comment:
Tutok nechyba tiez ten security?

Reply:
Myslím že úvahu som mal takú že netreba, lebo reálne len vybratím receptu sa nič nezačne, až štartom prvého kroku. Čiže sa nemôže stať že by sa napríklad zaplo ohrievanie, ani že by tým niekto posral rozrobené varenie.

A nemyslím že je pravdepodobné že by sme to nechávali len tak voľne pohodené zapnuté keď sa nejde variť.

čiže je to asi viac menej jedno by som povedal